owlbot.py

```diff
@@ -32,8 +32,6 @@
     intersphinx_dependencies={
         "pandas": "http://pandas.pydata.org/pandas-docs/stable/",
         "geopandas": "https://geopandas.org/",
-        "proto-plus": ("https://proto-plus-python.readthedocs.io/en/latest/", None),
-        "protobuf": ("https://googleapis.dev/python/protobuf/latest/", None),
```

Comment:
IIRC these were added these ack when we re-added legacy proto-based types to `v3`. We should thus retain them.